wls-gui-wahllokalsystem/src/composables/common/dateTimeFormatter.ts

```diff
@@ -91,13 +123,21 @@ export function useDateTimeFormatter() {
     }
   }
 
+  const parseDateString = (dateString: string) => {
```

Comment:
damit es einheitlich zu `getDateFromTimeString` ist
```suggestion
  const getDateFromDateString = (dateString: string) => {
```